character-quiz/index.ts

```diff
@@ -78,8 +78,8 @@ const loadCharacters = async (author: string) => {
 				rating,
 			] = line.split(',');
 
-			const characterNames = characterName.split(/[、&]/);
-			const characterRubys = characterRuby.split(/[、&]/);
+			const characterNames = characterName.split(/[、&]/).filter((name) => name !== '');
+			const characterRubys = characterRuby.split(/[、&]/).filter((name) => name !== '');
```

Comment:
```suggestion
			const characterNames = characterName.split(/[、&]/).filter((name) => name !== '');
			const characterRubys = characterRuby.split(/[、&]/).filter((name) => name !== '');
			
			if (characterNames.length === 0 || characterRubys.length === 0) {
				return null;
			}
```

にして、filterでnullを弾くようにしたい (型は type guard でなんとか)

Reply:
オリキャラもキャラクターではあるので、一旦は CharacterData にしてしまってあとから弾きたくない？